omnichain/multioutput/contracts/MultiOutput.sol

```diff
@@ -10,13 +10,15 @@ import "@openzeppelin/contracts/access/Ownable.sol";
 contract MultiOutput is zContract, Ownable {
     error SenderNotSystemContract();
     error NoAvailableTransfers();
+    error InvalidRecipient();
 
-    event DestinationRegistered(address);
-    event Withdrawal(address, uint256, address);
+    event Withdrawal(address, uint256, bytes);
 
-    address[] public destinationTokens;
     SystemContract public immutable systemContract;
 
+    uint256 constant BITCOIN = 18332;
+    address constant BITCOIN_ZRC20_ADDRESS = 0x65a45c57636f9BcCeD4fe193A602008578BcA90b;
```

Comment:
I think we can avoid hard-coding of the address by using:

```solidity
systemContract.gasCoinZRC20ByChainId(BITCOIN)
```

Similarly to:

https://github.com/zeta-chain/example-contracts/blob/6ba5da553aa531fe34524075cb7a505baaee132e/omnichain/staking/contracts/Staking.sol#L103